lib/diagnostic.rb

```diff
@@ -55,7 +78,24 @@ def example_params
   end
 
   describe 'POST create' do
-    # your test(s) here
+    def new_example
+      {
+        title: "title1",
+        content: "content1"
+      }
+    end
+    before(:each) do
+      post :create, example: new_example, format: :json
+    end
+
+    it 'is successful' do
+      expect(response.status).to eq(201)
+    end
+
+    it 'renders a JSON response' do
+      example_response = JSON.parse(response.body)
+      expect(example_response).not_to be_nil
+    end
   end
 end
```

Comment:
Yes, all fine, but FYI you were supposed to use the `example_params` that was define for you